tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/balo/readonly/SelectivelyImmutableTypeBaloTest.java

```diff
@@ -57,33 +57,33 @@ public void testImmutableTypesNegative() {
         int index = 0;
 
         // Assignment and initialization.
-        validateError(result, index++, "incompatible types: expected '(testorg/selectively_immutable:1.0" +
-                ".0:MixedRecord & readonly)', found 'testorg/selectively_immutable:1.0.0:MixedRecord'", 20, 38);
+        validateError(result, index++, "incompatible types: expected '(testorg_selectively_immutable:" +
```

Comment:
why `testorg/selectively_immutable` became `testorg_selectively_immutable` ?

Reply:
reverted in last commit